var/spack/repos/builtin/packages/umpire/package.py

```diff
@@ -128,6 +128,7 @@ class Umpire(CachedCMakePackage, CudaPackage, ROCmPackage):
     patch("camp_target_umpire_3.0.0.patch", when="@3.0.0")
     patch("cmake_version_check.patch", when="@4.1")
     patch("missing_header_for_numeric_limits.patch", when="@4.1:5.0.1")
+    patch("camp_resource.patch", when="@2022.10.0 %gcc@13 ^camp@2022.10.1")
```

Comment:
Do you know if other versions are affected by this? I'm guessing this could at least be applied independently of the camp version (GCC 13 is likely the trigger, as in #41408).

Reply:
So the build error is from mismatch between unmigrated code and new headers from compiler, with no relation to umpire, only not detected during install of camp. Thank you! I would move this to builtin/camp.